demisto_sdk/commands/common/tests/tools_test.py

```diff
@@ -2381,25 +2399,79 @@ def test_remove_tags_only(self):
         Then:
             - Text shouldn't have tags, but keep the text
         """
-        prefix = "<>"
-        suffix = "</>"
-        text = "some text <>tag text</>"
+        text = "some text <~>tag text</~>"
         expected_text = "some text tag text"
-        tag_parser = TagParser(prefix, suffix, remove_tag_text=False)
+        tag_parser = TagParser("")
         assert tag_parser.parse(text) == expected_text
 
 
 class TestMarketplaceTagParser:
     MARKETPLACE_TAG_PARSER = MarketplaceTagParser()
+    XSOAR_PREFIX = XSOAR_PREFIX_TAG
+    XSOAR_SUFFIX = XSOAR_SUFFIX_TAG
+    XSOAR_INLINE_PREFIX = XSOAR_INLINE_PREFIX_TAG
+    XSOAR_INLINE_SUFFIX = XSOAR_INLINE_SUFFIX_TAG
+
+    XSIAM_PREFIX = XSIAM_PREFIX_TAG
+    XSIAM_SUFFIX = XSIAM_SUFFIX_TAG
+    XSIAM_INLINE_PREFIX = XSIAM_INLINE_PREFIX_TAG
+    XSIAM_INLINE_SUFFIX = XSIAM_INLINE_SUFFIX_TAG
+
+    XPANSE_PREFIX = XPANSE_PREFIX_TAG
+    XPANSE_SUFFIX = XPANSE_SUFFIX_TAG
+    XPANSE_INLINE_PREFIX = XPANSE_INLINE_PREFIX_TAG
+    XPANSE_INLINE_SUFFIX = XPANSE_INLINE_SUFFIX_TAG
+
+    XSOAR_SAAS_PREFIX = XSOAR_SAAS_PREFIX_TAG
+    XSOAR_SAAS_SUFFIX = XSOAR_SAAS_SUFFIX_TAG
+    XSOAR_SAAS_INLINE_PREFIX = XSOAR_SAAS_INLINE_PREFIX_TAG
+    XSOAR_SAAS_INLINE_SUFFIX = XSOAR_SAAS_INLINE_SUFFIX_TAG
+
+    XSOAR_ON_PREM_PREFIX = XSOAR_ON_PREM_PREFIX_TAG
+    XSOAR_ON_PREM_SUFFIX = XSOAR_ON_PREM_SUFFIX_TAG
+    XSOAR_ON_PREM_INLINE_PREFIX = XSOAR_ON_PREM_INLINE_PREFIX_TAG
+    XSOAR_ON_PREM_INLINE_SUFFIX = XSOAR_ON_PREM_INLINE_SUFFIX_TAG
```

Comment:
If you need these values for the test, perhaps just set them here.

Reply:
didn't understand this one. where do you want the declaration of the values? 
I wanted to add all related code to the tag parser under this wrapper class. 
These values were once defined in the original tag parser class.
As a result of the refactoring to the tag parser, the values are no longer defined as part of the class. 
I wanted to keep the tests as similar as possible to what we had before the refactor thus the definitions here in the test class.